File: ordina/src/main/java/be/ordina/hp/pact/ordina/OrdinaApplicationConfig.java
package be.ordina.hp.pact.ordina;

import org.springframework.context.annotation.Bean;
import org.springframework.context.annotation.Configuration;
import org.springframework.web.client.RestTemplate;

@Configuration
public class OrdinaApplicationConfig {

    @Bean
    public RestTemplate restTemplate() {
        final RestTemplate restTemplate = new RestTemplate();
        return restTemplate;
    }
}

File: ordina/src/main/java/be/ordina/hp/pact/ordina/OrdinaApplicationConfig.kt
package be.ordina.hp.pact.ordina

import org.springframework.context.annotation.Bean
import org.springframework.context.annotation.Configuration
import org.springframework.web.client.RestTemplate

@Configuration
open class OrdinaApplicationConfig {

    @Bean
    open fun restTemplate(): RestTemplate {
        val restTemplate = RestTemplate()
        return restTemplate
    }
}
